easybuild/easyconfigs/o/OSU-Micro-Benchmarks/OSU-Micro-Benchmarks-5.7.1-gmpit-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'OSU-Micro-Benchmarks'
+version = '5.7.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://mvapich.cse.ohio-state.edu/benchmarks/'
+description = """OSU Micro-Benchmarks"""
+
+toolchain = {'name': 'gmpit', 'version': '2021a'}
+
+source_urls = ['https://mvapich.cse.ohio-state.edu/download/mvapich/']
+sources = [SOURCELOWER_TGZ]
+checksums = ['cb5ce4e2e68ed012d9952e96ef880a802058c87a1d840a2093b19bddc7faa165']
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('MPIwrapper', '2.3.2', versionsuffix),
+]
+
+configopts = 'CC="$MPICC" CXX="$MPICC"'
```

Comment:
Is this correct? I would have expected `CXX=$MPICXX`.

Reply:
Ha, you are right, that is a copy paste from an existing easyconfig, I will have to fix that.